File: archive/p/php/fizz-buzz.php
<?php

for ($i = 1; $i < 101; $i++)
{
  $output = "";

  if ($i % 3 == 0)
  {
    $output .= "Fizz";
  }

  if ($i % 5 == 0)
  {
    $output .= "Buzz";
  }

  if (!$output)
  {
    $output = $i;
  }

  echo $output . "\n";
}

File: archive/p/perl/fizz-buzz.pl
#!/usr/bin/env perl
use v5.42;

sub fizzbuzz ($n) {
    my $s = '';

    $s .= "Fizz" if $n % 3 == 0;
    $s .= "Buzz" if $n % 5 == 0;

    return $s || $n;
}

say fizzbuzz($_) for 1 .. 100;
